app/main.dev.ts

```diff
@@ -80,7 +92,21 @@ const createWindow = async () => {
           },
   });
 
-  void mainWindow.loadURL(`file://${__dirname}/app.html`);
+  if (process.platform === 'darwin') mainWindow.setTrafficLightPosition({ x: 10, y: 28 });
+
+  mainWindowState.manage(mainWindow);
+
+  if (process.env.NODE_ENV === 'development' && process.env.DEBUG_PROD !== 'true') {
+    void mainWindow.loadURL(`file://${__dirname}/app-dev.html`);
+  }
+
+  if (process.env.NODE_ENV === 'production' || process.env.DEBUG_PROD === 'true') {
+    void mainWindow.loadURL(`file://${__dirname}/app.html`);
+  }
```

Comment:
I find this to be a bit confusing. Shouldn't it be `if (ENV === 'development' || DEBUG_PROD === 'true') { ... } else { ... }`?

Reply:
When using debug true in either mode, I don't want to use the app-dev file.